mobile/src/main/java/org/openhab/habdroid/core/connection/AbstractConnection.java

```diff
@@ -10,15 +10,16 @@
 import org.openhab.habdroid.util.MyHttpClient;
 import org.openhab.habdroid.util.MySyncHttpClient;
 
+import java.io.IOException;
 import java.net.InetSocketAddress;
+import java.net.MalformedURLException;
 import java.net.Socket;
 import java.net.URL;
 
 public abstract class AbstractConnection implements Connection {
     private static final String TAG = AbstractConnection.class.getSimpleName();
-
+    private static final int MAX_CONNECT_TRIES = 10;
```

Comment:
`MAX_CONNECT_ATTEMPTS`

Reply:
IMHO this could also be local to `checkReachability` though, as the meaning of the constant is unclear outside of that method...one can't really derive from the constant name what its used for.